<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <column>
    <property name="text">
     <string>ID Отряда</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Назначение</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IDOtr</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>320</y>
     <width>203</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IDOtr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>349</y>
     <width>153</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IDOtr</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Назначение</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>379</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>409</y>
     <width>204</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>В главное меню</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
